app/code/Oleksandrr/Chat/view/frontend/templates/sidebar.phtml

```diff
@@ -7,7 +7,7 @@
      }}'
 >
     <h3><?= $block->escapeHtml(__('Chat with Admin')) ?></h3>
-
+    <?= $block->getBlockHtml('formkey') ?>
```

Comment:
@pandaUbuntu You already get this from cookies in the JS code. Why is this needed here? Form key must not be used in the cacheable content like this - otherwise all customers will get the same form key.